manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala

```diff
@@ -19,61 +19,140 @@
+    doc("Using", "query-using")
+    initQueries(
+      "CREATE INDEX ON :Scientist(name)",
+      "CREATE INDEX ON :Science(name)",
 
-  override def graphDescription = List(
-    "Andres:Swede KNOWS Peter",
-    "Stefan:German KNOWS Andres",
-    "Stefan KNOWS Peter",
-    "Emil:Expat KNOWS Peter",
-    "Emil KNOWS Andres",
-    "Emil KNOWS Stefan",
-    "Jim:Brit KNOWS Andres",
-    "Jim KNOWS Peter",
-    "Jim KNOWS Emil",
-    "Jim KNOWS Stefan"
-  )
-
-  override val setupConstraintQueries: List[String] = List("CREATE INDEX ON :Swede(surname)", "CREATE INDEX ON :German(surname)")
-
-  override val properties = Map(
-    "Andres" -> Map[String, Any]("age" -> 40l, "surname" -> "Taylor"),
-    "Peter" -> Map[String, Any]("age" -> 42l, "surname" -> "Neubauer"),
-    "Stefan" -> Map[String, Any]("age" -> 37l, "surname" -> "Plantikow"),
-    "Emil" -> Map[String, Any]("age" -> 37l, "surname" -> "Eifrem"),
-    "Jim" -> Map[String, Any]("age" -> 39l, "surname" -> "Webber")
-  )
-
-  def section = "Using"
-
-  @Test def query_using_single_index_hint() {
-    profileQuery(
-      title = "Query using an index hint",
-      text = "To query using an index hint, use +USING+ +INDEX+.",
-      queryText = "match (n:Swede) using index n:Swede(surname) where n.surname = 'Taylor' return n",
-      assertions = (p) => assert(p.toList === List(Map("n" -> node("Andres"))))
+      """CREATE
+        |(liskov:Scientist {name:'Liskov', born: 1939})-[:KNOWS]->(wing:Scientist {name:'Wing', born: 1956})-[:RESEARCHED]->(cs:Science {name:'Computer Science'})<-[:RESEARCHED]-(conway:Scientist {name: 'Conway', born: 1938}),
+        |(liskov)-[:RESEARCHED]->(cs),
+        |(wing)-[:RESEARCHED]->(:Science {name: 'Engineering'}),
+        |(chemistry:Science {name: 'Chemistry'})<-[:RESEARCHED]-(:Scientist {name: 'Curie', born: 1867}),
+        |(chemistry)<-[:RESEARCHED]-(:Scientist {name: 'Arden'}),
+        |(chemistry)<-[:RESEARCHED]-(:Scientist {name: 'Franklin'}),
+        |(chemistry)<-[:RESEARCHED]-(:Scientist {name: 'Harrison'})
+      """
     )
-  }
+    synopsis("The `USING` clause is used to influence the decisions of the planner when building an execution plan for a query.")
+    caution {
+      p("Forcing planner behavior is an advanced feature, and should be used with caution by experienced developers and/or database administrators only, as it may cause queries to perform poorly.")
+    }
+    section("Introduction") {
+      p("""When executing a query, Neo4j needs to decide where in the query graph to start matching.
+          |This is done by looking at the `MATCH` clause and the `WHERE` conditions and using that information to find useful indexes, or other starting points.""")
+      p("""The selected index might not be the best choice though -- sometimes multiple indexes could be used, and Neo4j
+          |has picked the wrong one (from a performance point of view), and sometimes (albeit rarely) it is better to not use an index at all.""")
```

Comment:
Agree with the note above. I would just make the following minor editorial change:
"better to not use an index" -> "better not to use an index"